src/devices/Card/CreditCard/TerminalTransactionQualifier.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    /// <summary>
+    /// Terminal Transaction Qualifier, capabilities and requirement
+    /// for the terminal to interact with the card.
+    /// This is used when asking for the Processing Data containing the 
+    /// details of the files to read to get card information
+    /// </summary>
+    [Flags]
+    public enum TerminalTransactionQualifier
+    {
+        // byte 1 byte 2 byte 3 byte 4
+        // byte 1
+        // bit 8: 1 = Mag-stripe mode supported 
+        MagStripeSupported = 0b1000_0000_0000_0000_0000_0000,
```

Comment:
description mentions 4 bytes but this has bits for 3

Reply:
Documentation is like that and the first is the one on the left. the last on the right. The last one on the right is always 0.

Reply:
And per the comment a bit before, I've changed for uint and made the last (first depends how you look at it) byte as 0 per the spec